develop-docs/api-server/application-domains/database-migrations/index.mdx

```diff
@@ -145,144 +145,202 @@ This is complicated due to our deploy process. When we deploy, we run migrations
 
 To avoid this, follow these steps:
 
-- Mark the column as nullable if it isn't, and create a migration. (ex. `BoundedIntegerField(null=True)`)
-- Deploy.
-- Remove the column from the model, but in the migration make sure we only mark the state as removed.
-- Deploy.
-- Finally, create a migration that deletes the column.
+- (Optional, but ideal) Make a PR to remove all uses of the column in the codebase in a separate PR. This mostly helps with code cleanliness. This should be merged ahead of the migration prs, but we don't need to worry about whether it is deployed first.
```

Comment:
I'm slightly confused about the optionality of this, is it just optional that it's a separate PR. It kind of makes it sound like we don't need to remove all the uses of the column, we definitely do.

I actually wouldn't mind if this just was the perscribed flow and remove the note about optionality.

```suggestion
- Make a PR to remove all uses of the column in the codebase in a separate PR. This mostly helps with code cleanliness. This should be merged ahead of the migration prs, but we don't need to worry about whether it is deployed first.
```

Reply:
Yeah I think I agree. The process is flexible for me, because maybe there are very few uses and you want to just do those in the first pr, but probably better to just be prescriptive